shell.nix

```diff
@@ -2,6 +2,9 @@
 with pkgs;
 mkShell {
   buildInputs = [
+    bash
```

Comment:
Won't hold it up but technically nix-shell requires bash in the first place so you shouldn't need to call it out directly.